src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList.tsx

```diff
@@ -24,11 +24,32 @@ const TaskList = ({ project, defaultClosed }: TaskListProps) => {
   const auth = useAuth();
   const [value, setValue] = useState<number>(1);
   const [addTask, setAddTask] = useState(false);
-
   const tasks = project.tasks;
```

Comment:
put the sort function at the top outside of the component and then sort it here (then you dont have to call the sort 3 times below)

Reply:
done